cohort-evaluator-spark/src/main/java/com/ibm/cohort/cql/library/HadoopBasedCqlLibraryProvider.java

```diff
@@ -0,0 +1,74 @@
+/*
+ * (C) Copyright IBM Corp. 2021
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+package com.ibm.cohort.cql.library;
+
+import java.io.IOException;
+import java.nio.charset.Charset;
+import java.util.Collection;
+import java.util.HashSet;
+import java.util.Set;
+
+import org.apache.commons.io.IOUtils;
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.FSDataInputStream;
+import org.apache.hadoop.fs.FileSystem;
+import org.apache.hadoop.fs.LocatedFileStatus;
+import org.apache.hadoop.fs.Path;
+import org.apache.hadoop.fs.RemoteIterator;
+
+public class HadoopBasedCqlLibraryProvider implements CqlLibraryProvider {
```

Comment:
No unit tests targeting this class specifically?

Reply:
Yea nothing crazy.  Just something that exercises all the code paths using local files in the project.